.github/workflows/ci_cb.yml

```diff
@@ -1,6 +1,10 @@
-name: Mirror and run GitLab CI
+name: CI run
 
-on: [push]
+on:
+  push:
+    branches:
+      - 'main'
+      - 'release/**'
```

Comment:
OK, so after this PR is merged, tests will not run after every push on every branch like we're used to. They'll only run once a PR is created. Is that correct?

Reply:
Yes, I avoid triggering the pipeline twice.